riscv/src/bootloader.rs

```diff
@@ -102,7 +105,7 @@ P0, P1, P2, P3 <== poseidon_gl(P0, P1, P2, P3, P4, P5, P6, P7, P8, P9, P10, P11)
 "#,
     );
 
-    for i in 0..merkle_tree_depth {
+    for i in 0..MERKLE_TREE_DEPTH {
```

Comment:
Can you write a comment showing the tree levels and what data is contained in each?

Reply:
No, more like showing how the bootloader uses Merkle trees. Like what the leaves are, which level corresponds to memory cells, which level corresponds to memory pages, etc